source/extensions/filters/network/redis_proxy/proxy_filter.h

```diff
@@ -99,7 +100,9 @@ class ProxyFilter : public Network::ReadFilter,
 
     // RedisProxy::CommandSplitter::SplitCallbacks
     bool connectionAllowed() override { return parent_.connectionAllowed(); }
-    void onAuth(const std::string& password) override { parent_.onAuth(*this, password); }
+    void onAuth(const std::string& username, const std::string& password) override {
```

Comment:
For most of the code, you're simply adding a username/password overload to the existing password only method. Why does this replace the password only method?

Reply:
Good point, I am going to rework this to be an overload